services/gemini-1/docker-compose-extra.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+volumes:
+  archival_node_data: {}
+
+services:
+  archival-node:
+    image: ghcr.io/subspace/node:${NODE_SNAPSHOT_TAG}
+    volumes:
+      - archival_node_data:/var/subspace:rw
+    restart: unless-stopped
+    ports:
+      - "30333:30333"
+    command: [
+      "--chain", "gemini-1",
+      "--base-path", "/var/subspace",
+      "--execution", "wasm",
+      "--pruning", "archive",
+      "--pool-kbytes", "51200",
+      "--listen-addr", "/ip4/0.0.0.0/tcp/30333",
+      "--rpc-cors", "all",
+      "--rpc-external",
```

Comment:
How port 9933 is used ?

Reply:
These nodes don't have RPC at all (or don't need to have at least)

Reply:
so rpc-external flag can be removed, correct ?